pkgs/servers/monitoring/prometheus/default.nix

```diff
@@ -1,97 +1,68 @@
 { stdenv
 , lib
 , go
-, pkgs
-, nodejs-14_x
-, nodePackages
 , buildGoModule
 , fetchFromGitHub
-, mkYarnPackage
+, yarn2nix-moretea
 , nixosTests
-, fetchpatch
 }:
 
 let
-  version = "2.30.3";
+  version = "2.32.1";
 
   src = fetchFromGitHub {
     rev = "v${version}";
     owner = "prometheus";
     repo = "prometheus";
-    sha256 = "1as6x5bsp7mxa4rp7jhyjlpcvzqm1zngnwvp73rc4rwhz8w8vm3k";
+    sha256 = "sha256-Yoj4Tf5lshq6p061I1+i8hYRrc7uRP2mfe/ueCVSo3I=";
   };
 
   goPackagePath = "github.com/prometheus/prometheus";
 
-  codemirrorNode = import ./webui/codemirror-promql {
-    inherit pkgs;
-    nodejs = nodejs-14_x;
-    inherit (stdenv.hostPlatform) system;
-  };
-  webuiNode = import ./webui/webui {
-    inherit pkgs;
-    nodejs = nodejs-14_x;
-    inherit (stdenv.hostPlatform) system;
-  };
+  webuiWorkspace = yarn2nix-moretea.mkYarnWorkspace {
```

Comment:
I think that breaks eval.

Reply:
It does, but I'm not sure why... I'll look into it some more when I run out of other ideas on how to build this.

Reply:
You are using IFD (importing json from a derivation) which is forbidden.

Reply:
Oh. Yep, should have looked at what `mkYarnWorkspace` is doing internally. My assumption was it just needs the lockfile... Guess we have to vendor the package.json too if we go this route.